### DISCUSSION TITLE
Would nosql database be supported in future?

### DISCUSSION THREAD

**Discussion by shoebsd31** [allaboutapps/go-starter#77] [Q&A] [answered]
I was wondering if there would be an ORM support for nosql databases like mongodb?

**Comment by majodev** [comment, marked as answer]
Hi @shoebsd31,

our backends mostly use `JSON` or `JSONB` data types for typical NoSQL use-cases, see [PostgreSQL JSON Types](https://www.postgresql.org/docs/13/datatype-json.html).

We only head to specialized NoSQL databases if we have actual requirements for them (<5% of our projects, e.g. time-series, caching) and most of the times, there are better alternative for those than MongoDB (e.g. [redis](https://github.com/redis/redis) for caching, [ClickHouse](https://github.com/ClickHouse/ClickHouse) / [TimescaleDB](https://github.com/timescale/timescaledb) for time-series).

To sum it up, we don't plan to include any dedicated NoSQL database adapters here.

Mario
